core/src/Kokkos_MathematicalFunctions.hpp

```diff
@@ -76,16 +77,17 @@ struct promote<float> {
 };
 template <class T>
 using promote_t = typename promote<T>::type;
-template <class T, class U>
+template <class T, class U,
+          bool = std::is_arithmetic<T>::value&& std::is_arithmetic<U>::value>
```

Comment:
```suggestion
          bool = std::is_arithmetic<T>::value && std::is_arithmetic<U>::value>
```

Reply:
I also dislike it but that is how Clang-Format wants it.